src/enzyme.js

```diff
@@ -0,0 +1,4 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
```

Comment:
This same configuration has been done already here: https://github.com/andela/indiana-ah-frontend/blob/ft-custom-footer-164369336/testSetup.js

Reply:
yea, that's okay. It would be corrected when I merge.

Reply:
Okay.